### DISCUSSION TITLE
Support for nanagement domain with Tanzu, perhaps limited Aria Suite components

### DISCUSSION THREAD

**Discussion by dwchan69** [vmware/powershell-module-for-vmware-cloud-foundation-power-management#96] [Community Help]
### Code of Conduct

- [X] I have read and agree to the project's Code of Conduct.
- [X] Vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue initial description to help the maintainers prioritize.
- [X] Do not leave "+1" or other comments that do not add relevant information or questions.
- [X] If you are interested in working on this issue or have submitted a pull request, please leave a comment.

### Description

Given VMware Holodeck VCF environment is built with Tanzu enable, would be great benefit to update supportability to include vSphere with Tanzu, with future prospect to handle Aria components.

### Use Case(s)

Shutdown and startup for VMware Holodeck

### Potential Configuration

NA

**Comment by tenthirtyam** [comment]
Please provide additional detail. Moving to the Discussions.

**Reply by dwchan69** [reply]
The current PS module can not power down a management domain created by VMware Holodeck\  It will error out upon execution stating it is "Tanuz enabled" - See attached inventory view below 

![image](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-power-management/assets/36751593/77144d7d-3e2c-4978-8720-6814b142024f)

**Comment by burnsjared0415** [comment]
so we would need to write the code to match this process:
https://docs.vmware.com/en/VMware-Cloud-Foundation/5.1/vcf-operations/GUID-38BA4D29-A7B0-4E52-9D83-F65AFD481465.html

**Comment by burnsjared0415** [comment]
the other issue is we would have to do a pause in the script to allow for customer to shutdown the containers in use then continue, which can be done just need to know how we want to design that thought process
